docs/README.md

```diff
@@ -175,23 +175,11 @@ See [Third-Party Commitizen Templates](third-party-commitizen.md).
 
 ## FAQ
```

Comment:
I think we could remove this section. as the content has been moved to FAQ page

Reply:
OK, should I also remove "Third-Party Commitizen Templates" and "Contributing" sections?
The content is also in other pages and only one link in README.